packages/core/src/extensions/Blocks/MultipleNodeSelection.ts

```diff
@@ -0,0 +1,100 @@
+import { Selection } from "prosemirror-state";
+import { Fragment, Node, Slice } from "prosemirror-model";
+import { Mappable } from "prosemirror-transform";
+
+/**
+ * This class represents an editor selection which spans multiple nodes/blocks. It's currently only used to allow users
+ * to drag multiple blocks at the same time.
```

Comment:
explain that it's based on the original selection + url.

(i.e.: what was your reasoning / design considerations?)